Update project name section (#7377)
microsoft/vscode-docs: remote/advancedcontainers/set-docker-compose-project-name.md

---
# Set Docker Compose project name

Visual Studio Code will respect the value you configure for the Docker Compose [project name](https://docs.docker.com/compose/project-name/).

> You can set the `COMPOSE_PROJECT_NAME` environment variable for the VS Code process, or specify it in a `.env` file in the same folder as the `docker-compose.yml`.

Author: @ntrogh I would also like to mention the name property, what do you think:

```suggestion
The top-level property `name` in the `docker-compose.yml` can be used to set the project name.

Alternatively, you can set the `COMPOSE_PROJECT_NAME` environment variable for the VS Code process, or specify it in a `.env` file in the same folder as the `docker-compose.yml`.
```

Author: I'd like to mention the `name` property in the `docker-compose.yml` because we only recently added support for that on our side and it is probably the easiest way to adjust the project name.

The part about the env variable still makes sense as another way of changing the project name when changing the `name` property is impractical for some reason and because that comes with the additional hurdle of having to restart VS Code.

The env variable use could be considered "advanced" now that we support `name`. We could remove it from the documentation to keep it simple. 🤔

Reviewer: Ah, now it makes sense. Your update looks good.

code .
```

> Or add the following to a `.env` file in the same folder as the `docker-compose.yml`:

Reviewer: ```suggestion
Alternatively, add the following entry to a `.env` file in the same folder as the `docker-compose.yml`:
```

```
COMPOSE_PROJECT_NAME=foo
```

> When no project name is configured and the `docker-compose.yml` is in the `.devcontainer` folder the Docker Compose default of using the `docker-compose.yml` folder's basename is overridden with `${project-folder-basename}_devcontainer` to avoid name collisions with other projects.

Reviewer: ```suggestion
When no project name is configured and the `docker-compose.yml` is in the `.devcontainer` folder, the Docker Compose default of using the `docker-compose.yml` folder's basename is overridden with `${project-folder-basename}_devcontainer` to avoid name collisions with other projects.
```
